unittest/python/crocoddyl_legacy/robots.py

```diff
@@ -4,7 +4,7 @@
 import pinocchio
 from pinocchio.robot_wrapper import RobotWrapper
 
-pinocchio.switchToNumpyMatrix()
+# pinocchio.switchToNumpyMatrix()
```

Comment:
Again revert all theses changes.
We don't want to modify them, they are static files to be erased.

Reply:
done

Reply:
done